contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -151,18 +154,141 @@ VueJs is a relatively new JavaScript framework as compared to Angular and React.
 2. **Language barriers:**<br/>
 A majority of users of VueJs are the Chinese as VueJs is developed by a Chinese American. He is supportive of the Chinese community and hence a lot of the existing plugins are written in Chinese. There might be some language barriers for an English speaking developer seeking for VueJs resources.
 
-#### Detailed Comparison of VueJs with other JavaScript frameworks can be found from:
+-----
+### Special features of VueJs
+Here are some features that VueJs differ from the other frameworks used:
+1. **Mutating of data in the DOM**
+
+In Vue, the state of the data can be directly modified. Let's say, there is a message in your app. To change the message, insert this line:
+```js
+// Vue way
+this.message = 'Hello Space';
+```
+
+In React, such a direct modification of data is not allowed, due to React's need to rerun lifecycle hooks after state is being updated. Data can only be updated using `this.setState` method.
+```js
+// React way
+this.setState({ message: 'Hello Space' });
+```
+<br>
+
+2. **2-way binding**: `v-model` is used to bind the DOM input field to its data variable. This effectively allows your DOM variables to be automatically in sync with your data, regardless of which one is being updated.
+In other words, this reduces the need for you to manually update your data.
+```js
+<input type="checkbox", v-model=isChecked">
+    <label for="checked">Select</label>
+```
+<br>
+
+3. **1-way data flow**: Data can only be passed from parent to child, via `props`. Props can be of any data type, including Objects.
```

Comment:
Also the props in Vue and React
Perhaps showing how Vue's is better than React (or the difference at least if Vue's not better than React?)